tests/python/unittest/test_operator.py

```diff
@@ -2002,9 +2002,10 @@ def test_depthwise_convolution():
 @with_seed()
 def test_convolution_independent_gradients():
     ctx = default_context()
-    # set a low bar for autotuned cudnn conv
-    atol = 1.0e-1 if ctx.device_type == "gpu" else 1.0e-3
-    rtol = 1.0e-2 if ctx.device_type == "gpu" else 1.0e-3
+    if ctx.device_type == "gpu":
```

Comment:
You can set `ctx = mx.cpu()` explicitly to avoid testing it on GPU. Please also add description or issue link so we can revisit it in the future.

Reply:
Makes sense. I will add some notes after filing an issue.